docs/core/deploying/native-aot/interop.md

```diff
@@ -47,20 +44,6 @@ On Windows, Native AOT uses a prepopulated list of direct P/Invoke methods that
 
 ### Linking
 
-To statically link against an unmanaged library, you need to specify `<NativeLibrary Include="filename" />` pointing to a `.lib` file on Windows and a `.a` file on Unix-like systems.
```

Comment:
This is required if one does DirectPInvoke on Windows. We need to link the import library lib file.

Reply:
Ah -- so since we don't support static linking, we should remove mention of "Direct P/Invoke" entirely?

Reply:
People are using direct p/invoke (you can do a quick search for DirectPInvoke on Github) and I can't think of a problem that direct p/invoke would cause. But we need to pass lib file of the import library to the linker on Windows so that it knows how to create the DLL import.

Reply:
DirectPInvoke is the preferable way to refer to external code in native code. It performs better and makes tools like dependency walker, or ldd work with our executables.